src/OpenTelemetry/Instrumentation/PropertyFetcher.cs

```diff
@@ -13,13 +13,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
+
 using System;
 using System.Linq;
 using System.Reflection;
 
-namespace OpenTelemetry.Instrumentation.Http.Implementation
+namespace OpenTelemetry.Instrumentation
```

Comment:
I know we've only ever used this in instrumentation, but given we're making this public and this functionality isn't necessarily specific to instrumentation does it make sense to put this in the root `OpenTelemetry` namespace?

Reply:
This *is* specific to instrumentation right? (Though this can be used for other purpose, the reason we are exposing it is to enable people to write own instrumentation for DS instrumented libraries)